sdk/python/.gitignore

```diff
@@ -0,0 +1,2 @@
+# generated by the build
+README.rst
```

Comment:
Given that everything in this folder is "generated by the build" - is there any specific need to `.gitignore` this file?  Won't it be unchanged unless there are corresponding README changes?

Reply:
I suppose there is no reason we couldn't check it in like we do for the other generated code.  I'll make that change.